src/Evolution/Executables/CurvedScalarWave/EvolveWorldtubeCurvedScalarWave.hpp

```diff
@@ -158,6 +163,28 @@ struct EvolutionMetavars {
       tmpl::list<::Events::Tags::ObserverMeshCompute<volume_dim>,
                  deriv_compute>;
 
+  template <size_t Number>
+  struct Axis : tt::ConformsTo<intrp::protocols::InterpolationTargetTag> {
+    static std::string name() { return "Axis" + std::to_string(Number); }
```

Comment:
It looks like you're observing Psi on these axes. Maybe change this name to be `PsiAlongNAxis` where N is either "X" or "Z". That way it's clear in the input file what's being observed.

Reply:
I changed it to `PsiAlongAxis` but since the axis can be specified in the input file to be any `LineSegment`, I'd rather not give them specific names so it does not look to be hard-coded. This way we also don't need a default case if `Number` is larger than 2.